recipes/tox/meta.yaml

```diff
@@ -36,17 +36,9 @@ about:
   license: MIT
   license_family: MIT
   license_file: LICENSE
-  summary: 'Tox is a generic virtualenv management and test command line tool you can use for
-    - checking your package installs correctly with different Python versions and interpreters
-    - running your tests in each of the environments, configuring your test tool of choice
-    - acting as a frontend to Continuous Integration servers, greatly reducing boilerplate
-      and merging CI and shell-based testing.'
+  summary: 'virtualenv-based automation of test activities'
   description: |
-    Tox is a generic virtualenv management and test command line tool you can use for
-    - checking your package installs correctly with different Python versions and interpreters
-    - running your tests in each of the environments, configuring your test tool of choice
-    - acting as a frontend to Continuous Integration servers, greatly reducing boilerplate
-      and merging CI and shell-based testing.
+    virtualenv-based automation of test activities
```

Comment:
Sorry should have been more clear. The `description` was fine as it was. Only the `summary` needed to change. That said, this can be fixed in the feedstock.